client/client.go

```diff
@@ -219,27 +256,26 @@ func (e ConnectionError) Unwrap() error {
 	return e.error
 }
 
-// raw performs a request and returns the resulting http.Response and
-// error you usually only need to call this directly if you expect the
-// response to not be JSON, otherwise you'd call Do(...) instead.
-func (client *Client) raw(ctx context.Context, method, urlpath string, query url.Values, headers map[string]string, body io.Reader) (*http.Response, error) {
+// raw creates an HTTP request, performs the request and returns an HTTP response
+// and error.
```

Comment:
I may need your help with this one Gustavo.

First, I take note of your feedback on how I can improve my commenting, thank you for that.

In this case, the design has changes so that the "other" comment part that says:

```
// ... you usually only need to call this directly if you expect the
// response to not be JSON, otherwise you'd call Do(...) instead.
```

I removed this because the comment is no longer possible. In the past the comment referred to the use case where the logs command directly called this to get access to the raw http body, but it is no longer possible to access this method using the Requester interface in the client, and the design has changed so that all use cases use ```Do()``` consistently, including the documented case above.

Maybe we should just remove the comment all-together as its not really adding any value as you rightfully points out ?